src/Catrobat/Controller/Web/MediaPackageController.php

```diff
@@ -56,6 +60,80 @@ public function mediaPackageAction(string $package_name, string $flavor, Transla
       throw $this->createNotFoundException('Unable to find Package entity.');
     }
 
+    $categories = $this->sortCategoriesFlavoredFirst($package->getCategories()->toArray(), $flavor, $translator);
+
+    $mediaDir = $this->getParameter('catrobat.mediapackage.path');
+
+    return $this->render('MediaLibrary/mediapackage.html.twig', [
+      'flavor' => $flavor,
+      'package' => $package_name,
+      'categories' => $categories,
+      'new_nav' => true,
+      'mediaDir' => '../../'.$mediaDir,
+    ]);
+  }
+
+  /**
+   * Searching the whole media library and returning results via twig. If a themed app is used (e.g. luna),
+   * MediaPackageFiles of that flavor plus all files of the standard flavor 'pocketcode'
+   * are returned. If the standard app is used, only files of the standard flavor 'pocketcode' are returned.
```

Comment:
is this the requested behaviour? @bernadettespieler

Reply:
This is they way the Media Library is alread working.

Try visiting https://share.catrob.at/app/media-library/looks first.

Try now visiting  https://share.catrob.at/luna/media-library/look which changes the flavor to luna. 

You now see a whole category "Luna&Cat" which will be shown for the Luna flavor exclusively.

By reusing as much as possible of the existing code in the Media Library (back and frontend) in order to have a working implementation ready as fast as possible this behavior is thus the same in the search function.

Reply:
It would be strange if you search in the media lib of pocket code and suddenly you see luna graphics you didn't have before. The search results should be "flavored" as well